### PR TITLE
Toggle soil probe

### PR DESCRIPTION
**Pull request by byrdman**
Fixes issue #50

### CODE REVIEW

File: wunderground-api.php [hunk 1]
```diff
@@ -35,17 +32,17 @@ function main() {
         $url .= "tempf=" . convertCelsiusToFahrenheit($row["AMBIENT_TEMPERATURE"]) . "&";
         $url .= "dewptf=" . calculateDewPointF($row["AMBIENT_TEMPERATURE"], $row["HUMIDITY"]) . "&";
         $url .= "baromin=" . convertMillibarsToInches(calculateMeanSeaLevelPressure($row["AIR_PRESSURE"], Settings::$stationElevationInMeters)) . "&";
-        $url .= "soiltempf=" . convertCelsiusToFahrenheit($row["GROUND_TEMPERATURE"]) . "&";
+        if (Settings::$soilTemperatureProbePresent == 1) {
```
Comment (JoshuaCarroll) [inline wunderground-api.php]: Datatype mismatch: The settings are read in as strings, so the line
`if (Settings::$soilTemperatureProbePresent == 1) {`
needs to be
`if (Settings::$soilTemperatureProbePresent == "1") {`

**Comment by JoshuaCarroll** [comment]
Side note, I also fixed some other errors in the setup.sh, so you might want to get latest before you submit.  Can you fix this pull request, or do you want to just submit a new one?

**Comment by byrdman** [comment]
Looks like I've cleaned it up now, take another gander at it.

**Comment by JoshuaCarroll** [comment]
It still shows there are conflicts. ?

**Comment by byrdman** [comment]
weird, thought I'd cleared those up.  I'll take another look.

**Comment by byrdman** [comment]
I redid the change and have submitted a new pull request

**Pull request closed without merging**